test/remote/gateways/remote_stripe_connect_test.rb

```diff
@@ -36,7 +36,7 @@ def test_successful_refund_with_application_fee
 
   def test_refund_partial_application_fee
     assert response = @gateway.purchase(@amount, @credit_card, @options.merge(:application_fee => 12))
-    assert refund = @gateway.refund(@amount-20, response.authorization, @options.merge(:refund_fee_amount => 10))
+    assert refund = @gateway.refund(@amount-20, response.authorization, @options.merge(:refund_fee_amount => "10"))
```

Comment:
Do we have a test for when `refund_fee_amount` is zero?

Reply:
Very good call, that revealed a sign error. Updated.